### PR TITLE
Omkar0114

### LINKED ISSUE #713: omkar
Screenshot 2021-09-14 17 25 59

### PR DESCRIPTION
**Pull request by Omkar0114**
resolves #713

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1324,5 +1324,11 @@
     "githubUsername": "Krishnapro",
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
-  }
+  },
+ {
```
Comment (schmelto) [inline data.json:1328]:
```suggestion
  {
```

File: data.json [hunk 2]
```diff
@@ -1324,5 +1324,11 @@
...
+    "name": "Omkar Kulkarni",
+    "githubUsername": "Omkar0114",
+    "imageUrl": "https://user-images.githubusercontent.com/88308267/133253323-504843cd-f744-4962-8c8a-3b44c76d7b87.png",
+    "issueNumber": 713
+}
```
Comment (schmelto) [inline data.json:1333]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json [hunk 3]
```diff
@@ -1324,5 +1324,11 @@
     "githubUsername": "Krishnapro",
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
-  }
+  },
+ {
+    "name": "Omkar Kulkarni",
+    "githubUsername": "Omkar0114",
+    "imageUrl": "https://user-images.githubusercontent.com/88308267/133253323-504843cd-f744-4962-8c8a-3b44c76d7b87.png",
```
Comment (Panquesito7) [inline data.json:1331]: I would recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/133314407-008ec664-bdbd-4428-a9c4-8c306f74a91c.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data.json (hunk 3 above)
Comment (Omkar0114) [reply data.json]: I updated link

File: data.json (hunk 2 above)
Comment (Omkar0114) [reply data.json]: removed extra spaces

**Comment by Omkar0114** [comment]
I updated link and removed extra spaces

File: data.json [hunk 4]
```diff
@@ -1324,5 +1324,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
-  }
+  },
+  {
+    "name": "Omkar Kulkarni",
+    "githubUsername": "Omkar0114",
+    "imageUrl": "https://user-images.githubusercontent.com/51391473/133314407-008ec664-bdbd-4428-a9c4-8c306f74a91c.png",
+    "issueNumber": 713
+  },
```
Comment (schmelto) [inline data.json:1333]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json (hunk 4 above)
Comment (Panquesito7) [inline data.json:1333]: Remove the comma here, it is not needed.
```suggestion
  }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by CodingSpecies** [comment]
@Omkar0114 please can you resolve the conflicts in the `data.json`, and also please do the changes explained by @Panquesito7. If you need any help feel free to ask. Thank you

File: data.json [hunk 5]
```diff
@@ -1325,6 +1325,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
   },
+  {
+    "name": "Omkar Kulkarni",
+    "githubUsername": "Omkar0114",
+    "imageUrl": "https://user-images.githubusercontent.com/51391473/133314407-008ec664-bdbd-4428-a9c4-8c306f74a91c.png",
+    "issueNumber": 713
+  }
```
Comment (Panquesito7) [inline data.json:1333]:
```suggestion
  },
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

**Pull request merged**